src/js/addComment.js

```diff
@@ -0,0 +1,24 @@
+import INVOLVEMENT_URL from './config.js';
+import displayComments from './displayComments.js';
+
+const formEl = document.querySelector('.add-comment');
+const { usernameEl, commentMsgEl } = formEl.elements;
+const addComment = () => {
+  formEl.addEventListener('submit', async (event) => {
+    event.preventDefault();
+
+    await fetch(`${INVOLVEMENT_URL}/comments`, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        item_id: formEl.dataset.item,
+        username: usernameEl.value,
+        comment: commentMsgEl.value,
+      }),
+    });
+
+    displayComments(formEl.dataset.item);
+  });
+};
```

Comment:
- Adding a comment to the selected popup is adding 2 same comments. KIndly make it right so adding a comment will only enter comment once.
- the input fields after submitting the comment become empty.
[Hint: Use usernameEl.value = '*empty string*' commentMsgEl.value = '*empty string*' after adding the comment]